plugins/chat/assets/javascripts/discourse/components/chat-message-actions-mobile.js

```diff
@@ -1,43 +1,49 @@
-import Component from "@ember/component";
+import Component from "@glimmer/component";
+import { getOwner } from "discourse-common/lib/get-owner";
+import { tracked } from "@glimmer/tracking";
 import discourseLater from "discourse-common/lib/later";
 import { action } from "@ember/object";
 import { isTesting } from "discourse-common/config/environment";
 
-export default Component.extend({
-  tagName: "",
-  hasExpandedReply: false,
-  messageActions: null,
+export default class ChatMessageActionsMobile extends Component {
+  @tracked hasExpandedReply = false;
+  @tracked showFadeIn = false;
 
-  didInsertElement() {
-    this._super(...arguments);
+  messageActions = null;
 
-    discourseLater(this._addFadeIn);
+  get capabilities() {
+    return getOwner(this).lookup("capabilities:main");
```

Comment:
Kind of wish there was a cool way of doing this like `@factory capabilities` or something, its weird that the inject here doesn't work for glimmer (maybe it will someday?)

https://github.com/discourse/discourse/blob/447da85c636dff5ba40a7abb141608fe7b4fa37b/app/assets/javascripts/discourse/app/pre-initializers/sniff-capabilities.js#L59-L59